src/playwright/transformPlaywright.test.ts

```diff
@@ -107,8 +101,9 @@ describe('Playwright', () => {
                 await testFn();
               } catch (err) {
                 if (err.toString().includes('Execution context was destroyed')) {
+                  console.log(\`An error occurred in the following story, most likely because of a navigation: "\${"Example/foo/bar"}/\${"A"}". Retrying...\`);
```

Comment:
@yannbf what kind of message is this?

Reply:
Most users who complained about the issue this PR is fixing experienced problems because their stories were doing navigation. This could help users understand why this happens. This is also the message that puppeteer/playwright provide if the context is destroyed.

It's possible that after retrying, the story still fails, so hopefully this helps the users

Reply:
I mean what does `"\${"Example/foo/bar"}/\${"A"}"` mean?

Reply:
It's a way to tell which story is failing in a more readable way, I guess we could use id instead. Or even build the story url if that makes sense.

Reply:
Do those strings get replaced by the actual title/name somehow?